readme/terminal.md

```diff
@@ -13,7 +13,7 @@ The notes can be [synchronised](#synchronisation) with various targets including
 Operating system | Method
 -----------------|----------------
 macOS            | `brew install joplin`
-Linux or Windows (via [WSL](https://msdn.microsoft.com/en-us/commandline/wsl/faq?f=255&MSPPError=-2147217396)) | **Important:** First, [install Node 8+](https://nodejs.org/en/download/package-manager/). Node 8 is LTS but not yet available everywhere so you might need to manually install it.<br/><br/>`NPM_CONFIG_PREFIX=~/.joplin-bin npm install -g joplin`<br/>`sudo ln -s ~/.joplin-bin/bin/joplin /usr/bin/joplin`<br><br>By default, the application binary will be installed under `~/.joplin-bin`. You may change this directory if needed. Alternatively, if your npm permissions are setup as described [here](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npms-default-directory-to-another-directory) (Option 2) then simply running `npm -g install joplin` would work.
+Linux or Windows (via [WSL](https://msdn.microsoft.com/en-us/commandline/wsl/faq?f=255&MSPPError=-2147217396)) | **Important:** First, [install Node 10+](https://nodejs.org/en/download/package-manager/). Node 8 is LTS but not yet available everywhere so you might need to manually install it.<br/><br/>`NPM_CONFIG_PREFIX=~/.joplin-bin npm install -g joplin`<br/>`sudo ln -s ~/.joplin-bin/bin/joplin /usr/bin/joplin`<br><br>By default, the application binary will be installed under `~/.joplin-bin`. You may change this directory if needed. Alternatively, if your npm permissions are setup as described [here](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npms-default-directory-to-another-directory) (Option 2) then simply running `npm -g install joplin` would work.
```

Comment:
I think we can remove the following sentence: `Node 8 is LTS but not yet available everywhere so you might need to manually install it.`.

@laurent22 what do you think? other than that the PR looks good and should be ready to be merged.

Reply:
Whoops, I thought I had removed that. Should we remove it entirely or change it to `Node 10 is LTS but not yet available everywhere so you might need to manually install it.`?

I'll take your recommendation here as I'm out of my element with the Node norms.

Reply:
IMO we should remove it completely. The latest LTS is 12.13.1, so node 10 is most likely available on all systems.
@laurent22 what do you think?

Reply:
I've gone ahead and removed the line, but can add it back if deemed necessary.